### ISSUE TITLE
[Feature Request]: Compatibility with Lora Block Weights

### ISSUE THREAD

**Issue by Neverdusk** [Bing-su/adetailer#184]
### Is your feature request related to a problem? Please describe.

I just started using this extension, and I noticed that it doesn't respect Lora Block Weights (https://github.com/hako-mikan/sd-webui-lora-block-weight) while performing fixes.

I first noticed the problem because I had a style LoRa in my text2img prompt, with a weight of 0.5 but a block weight of None.
E.g.: <lora:LoraName:0.5:NONE>.

Ordinarily, "NONE" is essentially equal to a weight of 0. I tend to swap between various weight presets (ALL, OUTD, INALL, etc), so I typically just use NONE when I want to deactivate this LoRa entirely. However, I noticed that ADetailer's results would seem to invoke this LoRa's style, despite the "NONE" preset. 

So I tested once again by changing the actual weight to 0.
E.g.: <lora:LoraName:0.0:NONE>.

Then the result with prefect clean, with no LoRa influence. 

While this isn't a major issue, it seems that ADetailer doesn't recognize Lora Block Weight presets, and will therefore ignore any weight presets - potentially changing the resulting style of any image that uses them. 

### Describe the solution you'd like

Ideally, ADetailer would somehow be able to read LoRa Block Weight presets (either only built-in ones or all user presets, including custom ones) and would maintain consistency with the prompt based on those presets. Then the results will mirror text/img, instead of forcing a non-preset LoRa weight onto the places affected by ADetailer.

### Describe alternatives you've considered

In my case, I could simply change the regular weight to 0, as I did in the above test. However, this wouldn't work for more complex LoRa Block Weight presets, such as OUTD, INS, INALL, etc.

**Comment by Bing-su** [comment]
There are some extensions that will always cause an error when run together, so I created a whitelist of extensions.
![Cap 2023-07-04 09-04-28-256](https://github.com/Bing-su/adetailer/assets/37621276/19bbfc8e-c036-4b79-814d-11cd1c0e29c4)
At the time I added this option, I was not aware of lora-block-weight, so it was not included. 

(Everyone, including me, needs to add `lora_block_weight` manually, since changing the option defaults doesn't update it).
